src/xrGame/ui/UIGameTutorial.cpp

```diff
@@ -97,8 +97,9 @@ bool CUISequenceItem::Stop(bool bForce)
 CUISequencer::CUISequencer() { m_flags.zero(); }
 void CUISequencer::Start(LPCSTR tutor_name)
 {
-    // Skip any tutorial except "game_loaded", since we need to show "st_press_any_key" hint
-    if (load_screen_renderer.IsActive() && xr_strcmp(tutor_name, "game_loaded") != 0)
+    // Skip any tutorial except "game_loaded" and "intro_game", on load screen
+    if (load_screen_renderer.IsActive() && xr_strcmp(tutor_name, "game_loaded") != 0 &&
```

Comment:
Should OR instead of AND
```suggestion
    if (load_screen_renderer.IsActive() && xr_strcmp(tutor_name, "game_loaded") != 0 ||
```

Reply:
Why? Ok. Assume that `load_screen_renderer.IsActive() == true` and `tutor_name == "game_loaded"`.

The first case:
`if (true && false && true)` returns false and tutorial "game_laoded" will start.

In your case:
`if (true && false || true)` returns true and the execution of the function is stopped.

Reply:
Oh! Sorry, you're right. I just mixed up `strcmp == 0` and `strcmp != 0` conditions.